src/main/java/com/coyoapp/tinytask/service/DefaultTaskService.java

```diff
@@ -41,6 +42,17 @@ private TaskResponse transformToResponse(Task task) {
     return mapperFacade.map(task, TaskResponse.class);
   }
 
+  @Override
+  @Transactional
+  public TaskResponse patchTask(String taskId, TaskRequestPatchDone taskRequestPatchDone) {
+    log.debug("patchTask(taskId={})", taskId);
+    Task task = getTaskOrThrowException(taskId);
+    Task patchedTask = mapperFacade.map(taskRequestPatchDone, Task.class);
+
+    task.setDone(patchedTask.getDone());
```

Comment:
you probably could getDone also from the TaskRequestPatchDone without mapping it first